lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -32,12 +33,22 @@ class GenerateCommand extends AbstractCommand
  */
 final class Version<version> extends AbstractMigration
 {
+    /**
+     * @param Schema $schema
```

Comment:
This adds no information to what is already expressed by native type. Please remove it.